MAINTAINERS.md

```diff
@@ -0,0 +1,79 @@
+# Maintainer Guidelines
+
```

Comment:
We might benefit from a sense of the Git-fu a maintainer should have. For those who aren't able to develop that level, is there a different workflow than the more advanced folks? Maybe a "Content Maintainer" would work through the Github interface only on existing material via the editing interface and the master branch, and a "Content Creator" would have more expectations around PRs, reviews, creation, branches, merging, etc.

Reading further, I think I see now the intent. I'm a little concerned that the iterative PR/review process is heavyweight enough to put some folks off of contributing. What might a process be for someone who has ideas for improvement, but doesn't have a Git background?